src/runtime/intern_.tt

```diff
@@ -0,0 +1,58 @@
+<#@ template debug="true" hostSpecific="true" #>
+<#@ output extension=".cs" #>
+<#
+    string[] internNames = new string[]

```

Comment:
Why use a template generator, when you can simply get `internNames` from reflection? It looks like unnecessary complication.

Reply:
Two reasons make this scenario happened.
1. You need to make sure the string is a c# intern string too to make code like this work without extra checking
```c#
const string s = "1";
switch (
    case "1":
        // both "1" point to the same string
       break;
```
2. If you need (1), you have to make the `internNames` set on compile.
Of course, if you don't (1), internNames can just set by using reflection from reading `PyIdentities`.